apache-jena-osgi/jena-osgi/src/main/java/org/apache/jena/osgi/Activator.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.jena.osgi;
+
+import org.apache.jena.system.JenaSubsystemRegistry;
+import org.apache.jena.system.JenaSubsystemRegistryBasic;
+import org.apache.jena.system.JenaSystem;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+public class Activator implements BundleActivator {
+	
+	/* 
+	 * Based on http://svn.apache.org/repos/asf/aries/trunk/spi-fly/spi-fly-examples/spi-fly-example-provider-consumer-bundle/src/main/java/org/apache/aries/spifly/pc/bundle/Activator.java
+	 * the Activator#start() waits for bundle extension by Aries SPI Fly, configures JenaSystem logging and requests for initialization.     
+	 */
+	public void start(BundleContext context) throws Exception {
+
+		Thread t = new Thread(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					Thread.sleep(500);
+				} catch (InterruptedException e) {
+				}				
+				setUpJena();
+			}
+		});
+		t.start();
+	}
+
+	private void setUpJena() {
+		JenaSubsystemRegistry r = new JenaSubsystemRegistryBasic() {
+			@Override
+			public void load() {
+				super.load();
+			}
+		};
```

Comment:
I agree that shading in `xercesImpl` is kind of… _shady_. (Hah!) But it would be worth making some effort to make plain to folks who have relied on it that they are going to have to make sure that it's available in their OSGi framework/container, just we don't have a bunch of people finding the same bundle resolution problem over and over.